specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json

```diff
@@ -10,6 +10,22 @@
   "consumes": [ "application/json" ],
   "produces": [ "application/json" ],
   "paths": {
+    "/providers/Microsoft.DevTestLab/operations": {
+      "get": {
+        "tags": [ "ProviderOperations" ],

```

Comment:
In all of the other specs I'm looking at, the tag is just `Operations` not `ProviderOperations`.

Reply:
`Operations` are already used in the end point for polling long running operations. I think `ProviderOperations` makes it clear.

Reply:
@ravbhatnagar, do you have thoughts on the name of the Operations group? It sounds like we're sorta flexible here, which is fine with me. I just want to make sure that's true though.

Reply:
@marstr - Do you mean the operationId? or the tags value. The operationId should be consistent between teams and I am not sure how tags is used. So I dont have a recommendation on that.

Reply:
I think its used by the autorest tool to segregate the generated code